sep490-frontend/src/app/modules/enterprise/services/popup.service.ts

```
@@ -3,9 +3,7 @@ import {SafeHtml} from '@angular/platform-browser';
 
 @Injectable()
 export class PopupService {
-
-  constructor() {
-  }
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)